src/drivers/St7789.h

```diff
@@ -1,10 +1,12 @@
 #pragma once
+#include "Spi.h"
```

Comment:
absolute paths please

```suggestion
#include "drivers/Spi.h"
```